src/main/kotlin/com/cognifide/gradle/aem/pkg/PrepareTask.kt

```diff
@@ -28,8 +31,17 @@ open class PrepareTask : AemDefaultTask() {
     fun prepare() {
         copyContentVaultFiles()
         copyMissingVaultFiles()
+        createEmptyJcrRoot();
```

Comment:
I dislike this corelation between `aemDownload` and `aemPrepare`. Why is it required?

Reply:
It is. Otherwise the shell package won't be a valid crx package.